### ISSUE TITLE
repo placement and naming

### ISSUE THREAD

**Issue by whyrusleeping** [noffle/fallback-ipfs-shell#1]
We can probably rename this to `go-ipfs-shell` and move it under the ipfs org.

**Comment by hackergrrl** [comment]
I can't believe I hadn't seen this issue until now!

Why do you suggest `go-ipfs-shell`? Should the default shell fall back to spawning its own? I think it'd be nice to have `go-ipfs-shell` be the repo that merely hosts the Shell interface, and have repos like `go-ipfs-api-shell` and `go-ipfs-ephemeral-shell`. What do you think @whyrusleeping?

**Comment by whyrusleeping** [comment]
Yeah, I agree with having a package for just the interface and then others
that implement it in different ways.

**Comment by whyrusleeping** [comment]
We should discuss with @lgierth what that interface should look like
